stacks/container-apps/augury-alarms.yml

```diff
@@ -0,0 +1,46 @@
+# stacks/container-apps/augury-alarms.yml
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Augury alarms
+Parameters:
+  OpsWarnMessagesSnsTopicArn:
+    Type: String
+  OpsErrorMessagesSnsTopicArn:
+    Type: String
+  OpsFatalMessagesSnsTopicArn:
+    Type: String
+  EnvironmentType:
+    Type: String
+  EnvironmentTypeAbbreviation:
+    Type: String
+  AuguryWorkerLogGroupName:
+    Type: String
+Resources:
+  AuguryActualsIngestJobInvocationMetric:
+    Type: "AWS::Logs::MetricFilter"
+    Properties:
+      FilterPattern: '[ImpressionsFetchJob] Success'
+      LogGroupName: !Ref AuguryWorkerLogGroupName
+      MetricTransformations:
+        - MetricName: !Sub "augury_actuals_rollups_ct_${EnvironmentTypeAbbreviation}"
+          MetricNamespace: LogMetrics
```

Comment:
I think it'd be better to put these metrics into their own namespace. `PRX/Augury` would be my vote.